README.md

```diff
@@ -54,6 +54,16 @@ if torch.xpu.is_available():
 torch_dtype = torch.float16 if device != "cpu" else torch.float32
 
 model = ParlerTTSForConditionalGeneration.from_pretrained("parler-tts/parler_tts_mini_v0.1").to(device, dtype=torch_dtype)
+
+# # Use with flash attention
```

Comment:
Ideally we can merge #65 first, and then have this PR isolate just the static cache changes

Reply:
Agree on that! I'll leave the merging of #65 to @ylacombe and then isolate the change